apps/netsuite/connector.py

```
@@ -76,11 +82,31 @@ def get_tax_code_name(item_id, tax_type, rate):
+        """
+        Checks if the sync is allowed
+
+        Returns:
+            bool: True
+        """
+        if attribute_count > SYNC_UPPER_LIMIT[attribute_type]:
+            workspace_created_at = Workspace.objects.get(id=self.workspace_id).created_at
+            if workspace_created_at > timezone.make_aware(datetime(2024, 10, 1), timezone.get_current_timezone()):
+                return False
+            else:
+                return True
```

Ruff

Return the negated condition directly

Inline condition

(SIM103)